pkgs/initiator/initiator_test.go

```diff
@@ -67,7 +73,7 @@ func TestStartDKG(t *testing.T) {
 	withdraw := common.HexToAddress("0x0000000000000000000000000000000000000009")
 	owner := common.HexToAddress("0x0000000000000000000000000000000000000007")
 	t.Run("happy flow", func(t *testing.T) {
-		intr, err := initiator.New(ops, logger, "v1.0.2")
+		intr, err := initiator.New(ops, logger, "v1.0.2", rootCert)
```

Comment:
@MatusKysel not sure its a good idea to have hardcoded version here, could be "test.version" to avoid confusion